daffodil-core/src/main/scala/org/apache/daffodil/core/grammar/primitives/PrimitivesDateTime.scala

```diff
@@ -201,18 +208,34 @@ abstract class ConvertTextCalendarPrimBase(e: ElementBase, guard: Boolean)
     p
   }
 
+  private lazy val dateTimeFormatterEv = {
+    val ev = new DateTimeFormatterEv(
+      calendarEv,
+      localeEv,
```

Comment:
This change may affect/fix https://issues.apache.org/jira/browse/DAFFODIL-2823 also, as the locale is being supplied differently now.